larray/core/array.py

```diff
@@ -5685,17 +5683,22 @@ def to_excel(self, filepath=None, sheet_name=None, position='A1',
                     if not os.path.isfile(filepath):
                         new_workbook = True
                     close = True
+            if overwrite_file:
```

Comment:
In that case, the test 'test_to_excel_xlwings' fails at line 
a2.to_excel(fpath, overwrite_file=True, engine='xlwings')
because the file already exists and new_workbook == False. 
What happens in that line is the creation a second sheet Sheet2 in which a2 is dumped.